deploy/patchfiles/1.5/add-NotifyLevel-column-and-default.sql

```diff
@@ -0,0 +1,24 @@
+if col_length(N'scheduler.Task',N'NotifyLevelEventlog') is null
+    alter table scheduler.Task add NotifyLevelEventlog int not null default 2
+go
+
+alter table scheduler.Task alter column NotifyLevelEventlog int not null;
+go
+
+if exists( select * 

```

Comment:
SQL2016 has been good to us
```sql
alter table scheduler.Task drop constraint if exists DF_Task_NotifyLevelEventlog 
```

Reply:
![](http://i3.kym-cdn.com/photos/images/newsfeed/000/620/434/f9e.gif)

Awesome to know. But the patchfile as written handles for system named defaults for those installations affected by line 2 :wink:

Reply:
Although it now occurs to me how obviously redundant line 5 is... :sweat_smile:

Reply:
The constraint name can be specified when adding a column directly, we should do that and simplify the overall script.